src/Sylius/Component/Core/Calculator/ProductVariantPricesCalculatorInterface.php

```diff
@@ -22,4 +22,6 @@ interface ProductVariantPricesCalculatorInterface extends ProductVariantPriceCal
      * @throws MissingChannelConfigurationException when price for given channel does not exist
      */
     public function calculateOriginal(ProductVariantInterface $productVariant, array $context): int;
+
+    public function calculateLowestPriceBeforeDiscount(ProductVariantInterface $productVariant, array $context): ?int;
```

Comment:
Adding a new method to the interface will be a BC break, extract this method to the new interface

Reply:
Or resolve it in the similar way as it is done in one of [the last PRs in RefundPlugin](https://github.com/Sylius/RefundPlugin/pull/409/files#diff-217282e0a3de891e1768f2413adc560887b87ff799b1f5dee191f1881e5edbefR21) from @coldic3, by adding a docblock above the interface